README.md

```diff
@@ -11,4 +11,8 @@ How to start server:
 
     $ jekyll serve
 
+You can write your blogs with future dated. Show future dated posts:
+
+    $ jekyll serve --future
```

Comment:
I've set publish date for blog post to Monday so in order to see it you need to run project locally with jekyll serve --future